README.md

```diff
@@ -1,12 +1,15 @@
 # Kythera
 
-![Github Actions][gha-badge]
-![MIT licensed][mit-badge]
-![APACHE V2 licensed][apache-badge]
+[![Github Actions][gha-badge][gha-url]
+[![MIT licensed][mit-badge][mit-url]
+[![APACHE V2 licensed][apache-badge][apache-url]
 
-[gha-badge]: https://img.shields.io/github/actions/workflow/status/polyphene/kythera/ci.yml?branch=master
+[gha-badge]: https://img.shields.io/github/actions/workflow/status/polyphene/kythera/ci.yml
```

Comment:
`master` was a typo, we're now using `main` and it seems redundant to refer the branch when it's the default.